Sources/xcprojectlint-package/CheckForWhitespaceSpecifications.swift

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2018 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import Foundation
+
+public func checkForWhiteSpaceSpecifications(_ project: Project, errorReporter: ErrorReporter) -> Int32 {
+  let mapToGroupError: (String, String) -> (String) -> String = { groupID, type in
```

Comment:
nitpick: `toGroupError` and `toFileError` reads a bit better at invocation `group.tabWidth.map(toGroupError(group.id, "tabWidth"))`.